notebooks/examples/CEMS.ipynb

```diff
@@ -0,0 +1,1130 @@
+{
```

Comment:
<p>This kind of iteration and aggregation is what Dask is designed to do automatically. If you point it at the top level of the dataset, it will do all this work for you seamlessly under the hood.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/catalyst-cooperative/pudl/pull/823/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='21'/>